gubernator/templates/filtered_log.html

```diff
@@ -13,11 +13,14 @@
 <input type="hidden" name="logfile" value="{{log_file}}">
 <input type="hidden" name="pod" value="{{pod}}">
 <input type="hidden" name="junit" value="{{junit}}">
+<hr>
+<input type="checkbox" name="wrap"{% if wrap %} checked{% endif %}> Line wrapping on
+<hr>
 <label><input type="checkbox" name="UID"{% if uid %} checked{% endif %}> Highlight Pod UID</label><br>
 <label><input type="checkbox" name="Namespace"{% if namespace %} checked{% endif %}> Highlight Namespace</label>
 </form>
 % if log
-<pre>
+{% if wrap %}<pre>{% else %}<pre class="nowrap">{% endif %}
```

Comment:
could also be `<pre{% if not wrap %} style="white-space:pre"{% endif %}>`. This toggle could also be done with a small piece of javascript.

I'd prefer not adding an option for wrapping. Maybe we can do something else to reduce the length of the lines? Right now every line in kubelet.log starts with a redundant 96 characters, which could be removed altogether.